src/Package/Impl/Repl/VsRInteractiveWorkflowProvider.cs

```diff
@@ -74,10 +79,20 @@ internal class VsRInteractiveWorkflowProvider : IRInteractiveWorkflowProvider, I
                                                     _plotsProvider, _activeTextViewTracker, _debuggerModeTracker, 
                                                     _shell, settings, _wss, () => DisposeInstance(sessionProvider));
             _disposableBag.Add(workflow);
+
+            sessionProvider.BrokerChanging += OnBrokerChanging;
             return workflow;
         }
 
+        private void OnBrokerChanging(object sender, EventArgs e) {
```

Comment:
It is the intent. If user is connecting, they most probably want REPL to come up and become active.